coursedata/texts/bg.yaml

```diff
@@ -27,6 +27,11 @@ ui:
     by: "от"
     start_quiz: "Започни теста"
 ClientErrorMessages:
+    Transpile_success1: "Good job!"
```

Comment:
We do not have to add these to the yaml, because we default to English anyway if there is no text in the yaml for a certain field